source/ranges.tex

```diff
@@ -1368,7 +1368,7 @@
       return to_address(ranges::begin(@\exposid{derived}@()));
     }
     constexpr auto data() const
-      requires range<const D> && contiguous_iterator<iterator_t<const D>> {
+      requires @\libconcept{range}@<const D> && contiguous_iterator<iterator_t<const D>> {
```

Comment:
```suggestion
      requires @\libconcept{range}@<const D> && @\libconcept{contiguous_iterator}<iterator_t<const D>> {
```
There's 3 uses of `contiguous_iterator` missing the change.

Reply:
fixed